### ISSUE TITLE
JOAN with Carla packaged version

### ISSUE THREAD

**Issue by StijnOosterlinck** [tud-hri/joan#30]
Hi, me again.

Is it possible to combine JOAN with a packaged version of Carla?

So instead of doing 'make launch' and pressing play in UE, I would run 'make package' as well and then use the resulting .exe file to run UE4Carla.

But I'm not sure if the state machine of JOAN still works well in this scenario? (for example: stopping the simulation, changing a setting - initialize - get ready - run again...)

Thanks in advance!

**Comment by OlgerSiebinga** [comment]
Yes, that will work fine. Just make sure that your packaged game starts with the correct map and game mode (JOAN game mode). 
Alternatively, you can use a startup script with the [command line command](https://docs.unrealengine.com/4.27/en-US/ProductionPipelines/CommandLineArguments/) to start the unreal editor.

**Comment by StijnOosterlinck** [comment]
Is there a reason that only the 'unreal editor' method is explained in the documentation?

Because I thought that making a packaged version could improve the performance, or am I wrong?

**Comment by OlgerSiebinga** [comment]
I believe this is just meant for debugging and quick evaluation during development. We have (indeed) seen better performance with packaged versions, but I've never measured this.

**Comment by StijnOosterlinck** [comment]
Makes sense, thanks for the quick replies!
